pkg/bitcoin/htlc/database/server.ts

```diff
@@ -0,0 +1,41 @@
+import  express  from "express"
+import mongoose  from "mongoose"
+import PostSchema, { PostschemaType } from "./model/db_schema"
+import { validateLocaleAndSetLanguage } from "typescript"
+import  bodyParser from 'body-parser'
+require("./model/db_schema")
```

Comment:
This is pretty hacky, please use a real "index.ts" file where you initialize your connection and server.